Code/Source/Lidar/PipelineGraph.cpp

```diff
@@ -170,12 +171,12 @@ namespace RGL
         SetIsFeatureEnabled(PipelineFeatureFlags::Noise, value);
     }
 
-    void PipelineGraph::Run()
+    void PipelineGraph::Run() const
```

Comment:
Making any "Run" method const may be suspicious - usually such method changes internal state of whatever is being run. Even if you are allowed to do it. How it works here?
+
Moreover, I had a discussion with Michał Pełka about const methods for classes accessed directly or indirectly through buses. This may be the case here, however I am not completely sure - please verify this. Michał stated, that in such cases const methods may cause compilation problems (different environment? compiler?).